source/routing/raptor_solution_reader.h

```diff
@@ -53,7 +53,24 @@ struct Dominates {
     }
 };
 
-typedef ParetoFront<Journey, Dominates> Solutions;
+struct ParetoFrontVisitor {
+    void at_dominated(const Journey& to_insert, const Journey& in_pool) {
+        log4cplus::Logger logger = log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("logger"));
```

Comment:
You might want to instantiate you logger in the instance, also please name them to help the reading later on

Reply:
Ok, the logger is now a member of the struct. https://github.com/CanalTP/navitia/pull/2937/commits/53cd072b2a5f6596daa177532ad871f67da15f12
What do you mean by "name them" ?